README.md

```diff
@@ -31,6 +31,8 @@ Run with: mvn verify
                             </goals>
                             <configuration>
                                 <projectName>cucumber-jvm-example</projectName>
+                                <!-- optional, defaults to false if not specified -->
+                                <skip>false</skip>
```

Comment:
this is also poor as I don't know what is skipped

Reply:
Most recent commit addresses your comment here. With that noted however, I posit that lines 44 and 45 are also unclear:
`<!-- optional, defaults to outputDirectory if not specified -->
<classificationDirectory>${project.build.directory}/classifications</classificationDirectory>`

Modern IDEs do display the parameter and property hints within the project under test's POM file... so is this really needed?